strawberry/relay/types.py

```diff
@@ -727,18 +723,13 @@ def resolve_connection(
         on `first`/`last`/`before`/`after` arguments.
 
         Args:
-            info:
-                The strawberry execution info resolve the type name from
-            nodes:
-                An iterable/iteretor of nodes to paginate
-            before:
-                Returns the items in the list that come before the specified cursor
-            after:
-                Returns the items in the list that come after the specified cursor
-            first:
-                Returns the first n items from the list
-            last:
-                Returns the items in the list that come after the specified cursor
+            info: The strawberry execution info resolve the type name from
+            nodes: An iterable/iteretor of nodes to paginate
+            before: Returns the items in the list that come before the specified cursor
+            after: Returns the items in the list that come after the specified cursor
+            first: Returns the first n items from the list
+            last: Returns the items in the list that come after the specified cursor
+            kwargs: Additional arguments passed to the resolver
```

Comment:
nit: okay, I start to feel bad, but now that I started I have to continue in the name of consistency(?) welp

```suggestion
            info: The strawberry execution info resolve the type name from.
            nodes: An iterable/iteretor of nodes to paginate.
            before: Returns the items in the list that come before the specified cursor.
            after: Returns the items in the list that come after the specified cursor.
            first: Returns the first n items from the list.
            last: Returns the items in the list that come after the specified cursor.
            kwargs: Additional arguments passed to the resolver.
```

Reply:
we might want to add a lint rule for this? if it exists?